src/main/java/config/DBConnection.java

```diff
@@ -1,29 +1,40 @@
 package config;
 
+import java.io.IOException;
+import java.io.InputStream;
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
+import java.util.Properties;
 
 public class DBConnection {
+
 	private static Connection con;
 
 	private DBConnection() {
-	};
-
+	}
+	
 	static {
 
+		ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+		InputStream input = classLoader.getResourceAsStream("application.properties");
+		Properties prop = new Properties();
+		try {
+			prop.load(input);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
 		try {
 
-			Class.forName(DatabaseConfig.DRIVER_NAME);
+			Class.forName(prop.getProperty("db.driver"));
 
 		} catch (Exception e) {
 			e.printStackTrace();
 		}
 
 		try {
-
-			con = DriverManager.getConnection(DatabaseConfig.CONNECTION_STRING, DatabaseConfig.DB_USER_NAME,
-					DatabaseConfig.DB_PASSWORD);
+			con = DriverManager.getConnection(prop.getProperty("db.host"), prop.getProperty("db.username"),

```

Comment:
1) The values are different for CONNECTION_STRING and db.host, did you checked running it in local after these changes?

2) Instead make a variable of connection String and then use it later,
The connection string contains the values in this format: 
CONNECTION_STRING = DB_HOST + ":" + DB_PORT + "/" + DB_NAME;

3) Rather than reading the properties value directly in DBConnection.java, is it possible to read the values in DatabaseConfig.java class itself and initialize the static variables, if yes then prefer that one.

Reply:
Tried to Implement same, Kindly review.